corehq/motech/dhis2/tests/test_entities_helpers.py

```diff
@@ -437,6 +440,122 @@ def test_existing_relationship_not_created_again(self):
         assert relationship_created.relationship_type_id != existing_relationship.relationship_type_id
 
 
+class TestRegisterTrackedEntityInstance(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain_obj = create_domain(DOMAIN)
+        cls.factory = CaseFactory(domain=DOMAIN)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain_obj.delete()
+        super().tearDownClass()
+
+    def setUp(self):
+        build_patch = patch('corehq.motech.dhis2.entities_helpers._build_tracked_entity')
```

Comment:
Just a sidenote. The `mock` library supports using `patch` as a decorator, which is a nice alternative